File: lecture_notes/08 collections/7-object.js
'use strict';

const ages = {
  'Vasia Pupkin': 19,
  'Marcus Aurelius': 1860,
};
console.log({ ages });

ages['Vasia Pupkin'] = 20;
console.log({ ages });

Reflect.deleteProperty(ages, 'Vasia Pupkin');
console.log({ ages });

console.log({
  'Vasia Pupkin': Reflect.has(ages, 'Vasia Pupkin'),
  'Marcus Aurelius': Reflect.has(ages, 'Marcus Aurelius'),
});

File: lecture_notes/08 collections/7-object.py
ages = {"Vasia Pupkin": 19, "Marcus Aurelius": 1860,}
print(f"ages: {ages}")

ages["Vasia Pupkin"] = 20
print(f"ages: {ages}")

ages.pop("Vasia Pupkin")
print(f"ages: {ages}")

print(f"Vasia Pupkin: {'Vasia Pupkin' in ages}, Marcus Aurelius: {'Marcus Aurelius' in ages}")
